types/v1/metrics.go

```diff
@@ -24,6 +26,29 @@ type VolumeMetrics struct {
 	Name              string  `json:"Name"`
 }
 
+// VolumeStats is used to store the collected metrics
+// TODO: Make this generic, so that it can be used by mayactl
+// and other components of maya.
+type VolumeStats struct {
+	Iqn                 string      `json:"iqn"`
+	Reads               json.Number `json:"ReadIOPS"`
+	TotalReadTime       json.Number `json:"TotalReadTime"`
+	TotalReadBlockCount json.Number `json:"TotalReadBlockCount"`
+	TotalReadBytes      json.Number `json:"TotalReadBytes"`
+
+	Writes               json.Number `json:"WriteIOPS"`
```

Comment:
json tag should also be changed to Writes instead of WriteIOPS.

Reply:
same comments as above